projects/observability/src/shared/dashboard/data/graphql/table/spans/spans-table-data-source.model.ts

```diff
@@ -15,6 +15,9 @@ import { TableDataSourceModel } from '../table-data-source.model';
   type: 'spans-table-data-source'
 })
 export class SpansTableDataSourceModel extends TableDataSourceModel {
+  // Mandatory fields: added to the request even if they are not visible
+  private mandatoryColumns: string[] = ['traceId'];
```

Comment:
This should be `readonly` since we aren't updating it.

Reply:
done